docs/examples.md

```diff
@@ -56,6 +56,11 @@ You can always switch this during the lifetime of the limiter:
     limiter.enabled = False
 ```
 
+#### Note: config file is set to ".env" by default. Therefore, if you have a .env file setup for something else already, specify the config_filename parameter
```

Comment:
This is the only section using 4# - - use use 2## to keep it consistent?
Consider moving this lower, to line 74, to keep it near another customization

Reply:
Also maybe use a shorter section title, like "Using the config file" and set your comment above as the paragraph content?